jedi_vim.py

```diff
@@ -157,10 +158,21 @@ def process(self):
         if not (self._process and self._process.poll() is None):
             cmd = os.path.join(
                 os.path.dirname(os.path.abspath(__file__)), self.remote_cmd)
+            # On Windows platform, need STARTUPINFO
+            if platform.system() == 'Windows':
+                si = subprocess.STARTUPINFO()
+                si.dwFlags = subprocess.STARTF_USESHOWWINDOW;
+                si.wShowWindow = subprocess.SW_HIDE;
+                stderr = subprocess.STDOUT
+            else:
+                si = None
+                stderr = subprocess.PIPE
```

Comment:
Redirecting stderr to PIPE require process.stderr to be read, or it may fill the pipe and block forever.
To preserve default behavior, set stderr to `None`.